webapp/src/components/InputField/index.tsx

```diff
@@ -1,70 +1,52 @@
-import * as React from 'react';
-import {FormattedMessage} from 'react-intl';
+import React, {ReactNode} from 'react';
 
 import {InputFieldType} from 'types';
 
 type Props = {
-    heading: {
-        id: string;
-        message: string;
-    };
+    heading: ReactNode;
```

Comment:
```suggestion
    heading: string | ReactNode;
```
Do this change everywhere where you have used only `ReactNode`.